metashare/api/models.py

```diff
@@ -165,14 +171,17 @@ class RepositorySlug(AbstractSlug):
 
 
 class Repository(
-    PopulateRepoIdMixin, HashIdMixin, TimestampsMixin, SlugMixin, models.Model,
+    PopulateRepoIdMixin, HashIdMixin, TimestampsMixin, SlugMixin, models.Model
 ):
     repo_owner = StringField()
     repo_name = StringField()
     name = StringField(unique=True)
     description = MarkdownField(blank=True, property_suffix="_markdown")
     is_managed = models.BooleanField(default=False)
     repo_id = models.IntegerField(null=True, blank=True, unique=True)
+    branch_name = models.CharField(
+        max_length=100, blank=True, null=True, validators=[validate_unicode_branch]
+    )
```

Comment:
If we're going to store this in the DB, it should probably default to `master`, and also be populated from the `repo.default_branch`.